<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="charactors"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>charactors</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="thread_nums"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>thread_nums</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="A_Z">
      <property name="text">
       <string>所有大写拉丁文_A-Z)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="a_z">
      <property name="text">
       <string>所有小写拉丁文(a-z)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="all_numbers">
      <property name="text">
       <string>所有数字</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="all_special_charactors">
      <property name="text">
       <string>所有特殊符号</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,0">
      <item>
       <widget class="QLineEdit" name="dir_show_line"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>选择文件夹</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="show_edit"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>开始暴力破解</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
